ui/package.json

```diff
@@ -60,7 +60,6 @@
     "@angular/compiler": "7.1.3",
     "@angular/core": "^5.2.2",
     "@angular/forms": "^5.2.2",
-    "@angular/http": "^5.2.2",
```

Comment:
This is now deprecated and duplicated as `@angular/common/http` - relevant code updated